docs/source/content.md

```diff
@@ -170,7 +170,8 @@ HTTP/1.1 404 Not Found
 Content-Type: application/json
 
 {
-  'error': 'NotFound'
+  'message': "Resource not found: https://<resource url>",
```

Comment:
JSON content-type has its keys and values surrounded by double quotes.

```suggestion
  "message": "Resource not found: https://<resource url>",
```

Reply:
@stevepiercy I forgot checking 
Sorry'! :(